stable/prometheus/values.yaml

```diff
@@ -684,6 +684,43 @@ serverFiles:
             target_label: __metrics_path__
             replacement: /api/v1/nodes/${1}/proxy/metrics
 
+
```

Comment:
Can we add some header comment here, why we need this.

Similar to https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml#L53 and https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml#L88